tfe/resource_tfe_team_access.go

```diff
@@ -128,7 +128,7 @@ func resourceTFETeamAccess() *schema.Resource {
 				ForceNew: true,
 				ValidateFunc: validation.StringMatch(
 					workspaceIdRegexp,
-					"must be the workspace's external_id",
+					"must be the workspace's immutable ID, which looks like ws-<RANDOM STRING>",
```

Comment:
For this and all references in documentation, I don't think beyond the context of this big migration anyone cares about the 'immutable' part of this description. How about...

```suggestion
					"must be a valid workspace ID (ws-*****)",
```

Reply:
that's part of why i changed it here. this validator function was only added because of the big migration and this is the way we refer to things in the documentation as well. i wasn't sure if these validator functions would really be that useful after the deprecation was completely done or not. 

i went ahead and changed it to this `must be a valid workspace ID (ws-<RANDOM STRING>)` to keep things consistent with our doc examples.